src/screens/OrganizationPeople/OrganizationPeople.tsx

```diff
@@ -28,9 +29,12 @@ function organizationPeople(): JSX.Element {
 
   document.title = t('title');
 
+  const location = useLocation();
+  const role = location?.state as any;
```

Comment:
Dont use `any` here

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/97614113/5ad33044-61f5-492c-b5d5-52628820bfa9)
removed the use of `any` defined `interface` instead